### PR TITLE
Bug 9441 Timeout popup appearing when user is being active.

### PR DESCRIPTION
**Pull request by nickwhm**
I've changed the functionality so it only has one version of the timeout functionality. Which is local to the unAuth chb app.

### CODE REVIEW

**Review by monalisasarkar123 — COMMENTED** [review]
take updated pull from main

**Review by monalisasarkar123 — COMMENTED** [review]
looks like the 0 Seconds counter functionality has broken.

File: src/samples/UnAuthChildBenefitsClaim/index.tsx [hunk 1]
```diff
@@ -49,6 +44,16 @@ function initTimeoutUnauth(setShowTimeoutModal) {
   }, milisecondsTilWarning);
 }
 
+// Sends 'ping' to pega to keep session alive and then initiates the timout
+function staySignedIn(setShowTimeoutModal, refreshSignin = true) {
+  if (refreshSignin) {
```
Comment (lcart97) [inline src/samples/UnAuthChildBenefitsClaim/index.tsx:49]: if refreshSignin  is always true, why do we need to have the if?

File: src/samples/UnAuthChildBenefitsClaim/index.tsx [hunk 2]
```diff
@@ -533,21 +522,13 @@ export default function UnAuthChildBenefitsClaim() {
...
-              bShowResolutionScreen
-            );
+            staySignedIn(setShowTimeoutModal);
           }}
           signoutHandler={() => {
             if (bShowResolutionScreen) {
               triggerLogout();
             } else {
               sessionStorage.setItem('hasSessionTimedOut', 'true');
-              clearTimer();
```
Comment (lcart97) [inline src/samples/UnAuthChildBenefitsClaim/index.tsx:550]: Why did you remove this?

Also there appears to be some overlap with: https://github.com/norm-l/odx/pull/706

File: src/samples/UnAuthChildBenefitsClaim/index.tsx (hunk 1 above)
Comment (nickwhm) [reply src/samples/UnAuthChildBenefitsClaim/index.tsx]: I've updated that, as yes it doesn't seem to be used anywhere.

File: src/samples/UnAuthChildBenefitsClaim/index.tsx (hunk 2 above)
Comment (nickwhm) [reply src/samples/UnAuthChildBenefitsClaim/index.tsx:550]: I removed this as we are no longer using the timeOutUtils and clearTimeout is from there. The main issue here was we had two timeouts running at the same time. 

Until every journey has the exact same timeout functionality we can't move towards having one version of the timeout at the moment.

Comment (nickwhm) [reply src/samples/UnAuthChildBenefitsClaim/index.tsx:550]: There is overlap. I have put https://github.com/norm-l/odx/pull/706 into draft until I can merge main back into it after this has been merged.

**Comment by nickwhm** [comment]
Have created new branch for this from main
